plugins.json

```
@@ -1647,6 +1647,48 @@
     "enterprise": false,
     "updated_at": "2019-06-12T19:04:44Z"
   },
+  {
+    "homepage_url": "https://github.com/mattermost/mattermost-plugin-todo",
+    "icon_data": "",
+    "download_url": "https://github.com/mattermost/mattermost-plugin-todo/releases/download/v0.3.0/com.mattermost.plugin-todo-0.3.0.tar.gz",
+    "release_notes_url": "https://github.com/mattermost/mattermost-plugin-todo/releases/tag/v0.3.0",
```

When I run `go run ./cmd/generator/ add mattermost-plugin-todo v0.3.0 --beta --official` locally, I get a different outcome for these lines:
```suggestion
    "download_url": "https://plugins-store.test.mattermost.com/release/mattermost-plugin-todo-v0.3.0.tar.gz",
    "release_notes_url": "",
```

Did you generate all of these changes using the same command as above? I curious to find out why my generated file is difference then yours.